mk/libsodium-src.mk

```diff
@@ -1,6 +1,7 @@
-LIBSODIUM_VERSION := 1.0.14
+LIBSODIUM_TAG	  := 1.0.18-RELEASE
+LIBSODIUM_VERSION := 1.0.18
```

Comment:
`LIBSODIUM_VERSION` is replaced by `LIBSODIUM_TAG` so we can delete it?

Reply:
```suggestion
```